### PR TITLE
Move filters to config

### PR DESCRIPTION
**Pull request by rubenpeters91**
### Fixed
- admin dashboard now also shows clinics outside of RCT
- moved sql filters to config

### Changed
- clinic_name used in dashboard now refers to TeleQ name
- updated requirements

Closes: #70

### CODE REVIEW

File: data/raw/data_prediction.sql [hunk 1]
```diff
@@ -54,109 +56,25 @@ FROM [PUB].[no_show].[HealthcareService] SUBAGENDA JOIN [PUB].[no_show].[Healthc
...
-        'ZH0192', 'ZH0561', 'Z01316', 'Z01188', 'Z01234'
-    )
     AND HOOFDAGENDA.identifier_system = 'https://metadata.umcutrecht.nl/ids/HixAgenda'
     AND HOOFDAGENDA.active = 1
-    AND (
-        HOOFDAGENDA.identifier_value IN (
+    AND HOOFDAGENDA.identifier_value IN (
+            -- Revalidatie en sport
+            'ZH0307',  -- RF&S Revalidatiegeneeskunde
+            'ZH0435',  -- RF&S Sportgeneeskunde
```
Comment (WoltersEric) [inline data/raw/data_prediction.sql:64]: We'll probably remove this when every poli is included right? Currently we still need to update both the config and query if we add more poli's

**Review by WoltersEric — COMMENTED** [review]
Looks good to me! I can't find the change that causes the mismatch in resulting appointments. Won't approve until we find that out

File: data/raw/data_prediction.sql (hunk 1 above)
Comment (rubenpeters91) [reply data/raw/data_prediction.sql:64]: Yes, but for now I want to reduce the size of the data with only this filter

**Review by WoltersEric — APPROVED** [review]
Looks good to me!

**Pull request merged**